content/directory/Y-Designs.md

```diff
@@ -1,17 +0,0 @@
----
```

Comment:
Boof. Really wanted to talk with these people. How were you able to find out? I'm curious to find good avenues for hearing about opens & closures for the future.